stock_release_channel/models/stock_release_channel.py

```diff
@@ -270,6 +274,13 @@ def _compute_is_release_allowed(self):
         for rec in self:
             rec.is_release_allowed = rec.state == "open" and not rec.release_forbidden
 
+    @api.depends("state")
+    def _compute_is_action_safe_sleep_allowed(self):
+        for rec in self:
+            rec.is_action_safe_sleep_allowed = bool(
+                rec.state in ["locked", "open"] and rec.open_picking_ids
+            )
+
```

Comment:
Isn't it possible to use `is_action_sleep_allowed`?
If not `state in ["locked", "open"]` could move into a own method to be reused?

```suggestion
    @api.depends("state", "is_action_sleep_allowed")
    def _compute_is_action_safe_sleep_allowed(self):
        for rec in self:
            rec.is_action_safe_sleep_allowed = bool(
                rec.is_action_sleep_allowed and rec.open_picking_ids
            )

```

Reply:
> If not could we move `state in ["locked", "open"]` into an own method to be reused in both compute methods?

@mt-software-de In fact, this is too broad as if channel is delivered but not asleep, button is still visible but has no interest as it should be when outgoing transfers are not done/cancel